public/app/plugins/datasource/prometheus/querybuilder/components/metrics-modal/styles.ts

```diff
@@ -0,0 +1,122 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = (theme: GrafanaTheme2, disableTextWrap: boolean) => {
+  return {
+    modal: css`
+      width: 85vw;
+      ${theme.breakpoints.down('md')} {
+        width: 100%;
+      }
+      ${theme.breakpoints.up('xl')} {
+        width: 60%;
+      }
+    `,
+    inputWrapper: css`
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      gap: ${theme.spacing(2)};
+      margin-bottom: ${theme.spacing(2)};
+    `,
+    inputItemFirst: css`
+      flex-basis: 40%;
+    `,
+    inputItem: css`
+      flex-grow: 1;
+      flex-basis: 20%;
+      ${theme.breakpoints.down('md')} {
+        min-width: 100%;
+      }
+    `,
+    selectWrapper: css`
+      margin-bottom: ${theme.spacing(1)};
+    `,
+    selectItem: css`
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+    `,
+    selectItemLabel: css`
+      margin: 0 0 0 ${theme.spacing(1)};
+      align-self: center;
+      color: ${theme.colors.text.secondary};
+    `,
+    resultsHeading: css`
+      margin: 0 0 0 0;
+    `,
+    resultsData: css`
+      margin: 0 0 ${theme.spacing(1)} 0;
+    `,
+    resultsDataCount: css`
+      margin: 0;
+    `,
+    resultsDataFiltered: css`
+      margin: 0;
+      color: ${theme.colors.warning.text};
+    `,
+    alphabetRow: css`
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      align-items: center;
+      column-gap: ${theme.spacing(1)};
+      margin-bottom: ${theme.spacing(1)};
+    `,
+    alphabetRowToggles: css`
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      flex-wrap: wrap;
+      column-gap: ${theme.spacing(1)};
+    `,
+    results: css`
```

Comment:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/109082771/235498578-98eba1d7-c8ee-4f59-ab09-2ee8e29fbb16.png">
I want it taller still!

<img width="1398" alt="image" src="https://user-images.githubusercontent.com/109082771/235499091-64d90d0f-b824-44cd-8194-fec0a17d4a9e.png">
Looks like if you remove the hard coded height of 250px (with something like (`calc(80vh - 280px)` where `280px` is however much vertical space everything else in the component needs) , and instead define this height and the `max-height:80vh` (instead of 80%) of the top-level modal component in terms of vh units then we always use the middle 80% of the users monitor, instead of leaving a bunch of real estate on the table.